tests/base.py

```diff
@@ -44,6 +44,31 @@ def has_roce_hw_bug(vendor_id, vendor_part_id):
 
 
 class PyverbsAPITestCase(unittest.TestCase):
+    """
+    A base class for test cases which provides the option for user parameters.
+    These can be provided by manually adding the test case to the runner:
+    suite = unittest.TestSuite()
+    ... # Regular auto-detection of test cases, no parameters used.
+    # Now follows your manual addition of test cases e.g:
+    suite.addTest(PyverbsAPITestCase.parametrize(<TestCaseName>, dev_name='..'))
```

Comment:
This comment isn't relevant is it?

Reply:
This is related to the use of parametrize(), I've decided to drop it from this patchset in v2.
I think with the use of the new command-line parser we can drop parametrize() from RDMATestCase too?